rslib/Cargo.toml

```diff
@@ -38,6 +38,7 @@ wiremock.workspace = true
 
 [dependencies]
 criterion = { workspace = true, optional = true }
+pbkdf2 = { version = "0.12", features = ["simple"] }
```

Comment:
Please move this into the workspace Cargo.toml and reference it here, like the other deps.